getting_started/install_openshift.adoc

```diff
@@ -132,6 +132,11 @@ This command tells your RHEL system that the tools required to install
 {product-title} will be available from these repositories. Now we need the
 {product-title} installer that is based on Ansible.
 
+[NOTE]
+Older versions of {product-title} 3.11 supported only Ansible 2.6, the most
+recent versions of the playbooks have been updated to support Ansible 2.8 and
```

Comment:
If you're ok with this text, I'd open a follow-up to use this note instead:
> Older versions of {product-title} 3.11 supported only Ansible 2.6. The most
recent versions of the playbooks now support Ansible 2.8, which is the
preferred version to use.

Reply:
that's fine with me